evm/deploy-its.js

```diff
@@ -36,9 +35,13 @@ const { Command, Option } = require('commander');
  */
 
 async function deployImplementation(config, wallet, chain, options) {
-    const { env, salt, factorySalt, deployMethod, skipExisting, verify, yes } = options;
+    const { env, artifactPath, salt, factorySalt, deployMethod, skipExisting, verify, yes } = options;
     const verifyOptions = verify ? { env, chain: chain.name, only: verify === 'only' } : null;
 
+    const InterchainTokenService = artifactPath
+        ? getContractJSON('InterchainTokenService', artifactPath)
+        : getContractJSON('InterchainTokenService');
```

Comment:
```suggestion
    const InterchainTokenService = getContractJSON('InterchainTokenService', artifactPath);
```
If `artifactPath` isn't defined, the function should ignore it i think

Reply:
very true